app/Controllers/AuthController.js

```diff
@@ -31,8 +37,18 @@ async function profile(req, res) {
   res.send(deleteKeysFromObject(['roleId', 'password', 'createdAt', 'updatedAt'], req.user.toJSON()));
 }
 
+async function deleteUser(req, res) {
+  // get userId from request
+  const userId = req.user.id;
```

Comment:
shouldn't we add a security check so that a user have to send also his password?

Reply:
will be added in another PR